src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java

```diff
@@ -477,14 +477,14 @@ public static boolean containsGroup(JsonNode json, String group) {
           repo.setAdminGroup(project.getProjectAdminGroup());
           repo.setUserGroup(project.getProjectUserGroup());
         } else {
-          repo.setAdminGroup(defaultUserGroup);
+          repo.setAdminGroup(defaultAdminGroup);
```

Comment:
This seems to not be covered with a unit test. Can you please extends it?

Reply:
Done